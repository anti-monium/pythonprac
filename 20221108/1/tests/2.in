a = DivStr('zxcvbnhfgtrhehois')
print(*a//2)
for s in a // 2:
	print(s % 3)
b = DivStr('aoaoaooa mmmmmmmm')
for s in a // 2:
	for l in b // 3:
		print((s + l) % 5)
