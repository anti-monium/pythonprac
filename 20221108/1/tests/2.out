zxcvbnhf gtrhehoi
hf
oi
aoa
a m
mmm
aoa
a m
mmm
